kin-sdk-core/src/main/java/kin/sdk/core/util/HexUtils.java

```diff
@@ -0,0 +1,14 @@
+package kin.sdk.core.util;
+
+public class HexUtils {
+
+    public static byte[] hexStringToByteArray(String s) {
```

Comment:
isn't it the same like geth `Address.getBytes()`?
HexUtils shouldn't be public (no meaning to sdk client)

Reply:
No, the private key can't be initialized with Address object from geth, it's not 40chars long.
So we need our method, also it's a util so I don't mind to set it to public, @berryve wdyt ?

Reply:
ah, ok. :+1:
I think we shouldn't expose anything not valuable to client user

Reply:
Don't think that exposing this is problematic/confusing and it is better 'order' to have it within the util package.